### PR TITLE
add external etcd for e2e test

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>

### CODE REVIEW

File: test/bin/setup-e2e.sh [hunk 1]
```diff
@@ -46,11 +53,32 @@ for i in $(seq 1 "${number}"); do
...
+  etcd_size=$(${KUBECTL} -n ${etcdns} get statefulset.apps/etcd -o jsonpath='{.spec.replicas}')
+  etcd_services=http://etcd-0.etcd.${etcdns}:2379
+  for((i=1;i<$etcd_size;i++))
+  do
+    etcd_services=${etcd_services}",http://etcd-"$i".etcd.${etcdns}:2379"
+  done
+  sed -i "s@http://127.0.0.1:2379@${etcd_services}@g" $deploy_dir/external-etcd-patch.yaml
+  sed -i "s/--etcd-prefix=.*/--etcd-prefix=${namespace}\"/g" $deploy_dir/external-etcd-patch.yaml
+  sed -i "s/API_HOST/${external_host_ip}/g" $deploy_dir/external-etcd-patch.yaml
+
```
Comment (clyang82) [inline test/bin/setup-e2e.sh:81]: Can we setup a separated case to test external etcd? I mean we test 2 cases: external etcd and embedded etcd.

Comment (yanmxa) [reply test/bin/setup-e2e.sh:81]: For now, we use the external etcd in the e2e test and use the embedded etcd with the integration test.

So there is a scenario for external etcd in the tests.

Comment (clyang82) [reply test/bin/setup-e2e.sh:81]: sound good. thanks.

**Pull request merged**